pkg/syncer/cnsoperator/controller/cnsregistervolume/cnsregistervolume_controller.go

```diff
@@ -201,6 +201,18 @@ func (r *ReconcileCnsRegisterVolume) Reconcile(request reconcile.Request) (recon
 	log.Debugf("CNS Volume create spec is: %+v", createSpec)
 	vol, err := r.volumeManager.CreateVolume(ctx, createSpec)
 	if err != nil {
+		// volumeManager.CreateVolume already handles DiskURLPath which is already
```

Comment:
I did not get what this means. How does volumeManager.CreateVolume handles already registered diskUrlPath?